app/controllers/project_managers_controller.rb

```diff
@@ -0,0 +1,50 @@
+class ProjectManagersController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_project_manager, only: %i[ show edit update destroy ]
+
+  # GET /project_managers or /project_managers.json
+  def index
+    @project_managers = ProjectManager.all
+  end
+
+  # GET /project_managers/1 or /project_managers/1.json
+  def show
+  end
+
+  def edit
+  end
+
+  # PATCH/PUT /project_managers/1 or /project_managers/1.json
+  def update
```

Comment:
In real life, probably there would be `before_filter` actions to ensure a PM account can only be edited by that PM, but the requirements didn't mention it 👀